app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Here is a list of all posts from user 1 and bio</h1>
\ No newline at end of file
+<article class="main-article">
+  <div class="user-info-wrapper">
+    <%= image_tag @user.photo, :class => "user-avatar" %>
+    <div class="user-details">
+      <p><%= @user.name %></p>
+      <span>Number of posts: <%= @user.post_counter %></span>
+    </div>
+  </div>
+  
+  <div class="bio">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div class="post">
+      <h4>Post #<%= index + 1 %></h4>
+      <div>
+        <p class="post-title"><%= post.title %></p>
```

Comment:
- [ ] [Optional] Well! As I mentioned previously, you are showing the project title here. Considering that the title might be longer and the description is typically shorter in font size, I recommend displaying the post description here instead of the title for a more visually balanced and informative representation.